### PR TITLE
Product list

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
Added productrs page:
- added throttling to wishlist and cart buttons
- will add pop up let's in the later PR so please ignore for now
- will remove the mock be link from nav bar in final Stage so please ignore for now
- added sort and filter functions

### CODE REVIEW

**Review by tejas-muthya37 — APPROVED** [review]
3 things I liked about your code. 

1) Use of interface.
2) Use of BASE_IMG_URL & BASE_API_URL
3) Level of detailing in CSS.

File: public/index.css [hunk 1]
```diff
@@ -5,6 +5,11 @@
   max-width: 100vw;
   min-width: 100vw;
 }
+input,
+a,
+button {
+  cursor: pointer;
+}
```
Comment (samyakshah3008) [inline public/index.css:12]: You can avoid giving styling directly to element

**Review by samyakshah3008 — APPROVED** [review]
Clean code 
You can attach image or gif in PR description 
Added some single comment as suggestion 
Good folder structure 

Rest is all good for approval ✅

**Comment by SJTGSHIVAM** [comment]
@samyakshah3008 I have provided the live preview. if you missed it. its here https://app.netlify.com/sites/log2-mock/deploys/62485221f1d02d00092ff1e4 this is why I am not posting images.

File: public/index.css (hunk 1 above)
Comment (SJTGSHIVAM) [reply public/index.css:12]: its has to be a part of reset CSS. Will Fix it.

**Review by PeeyushPrashant — APPROVED** [review]
code looks clean and modular
good naming convention

**Review by Hnh013 — APPROVED** [review]
clean code , well structured , appreciable use of constants wherever required and good handling of states .

**Pull request merged**